rfcs/20211207-softmax-v2/README.md

```diff
@@ -0,0 +1,144 @@
+# RFC: Softmax v2
+
+## Introduction
+
+Several primitives in the library were designed with operation descriptors containing only `data_desc` member, which covers source and destination memory descriptors.
+There was an assumption that such operations do not change neither memory format layout through execution nor require a different output data type. And would never be.
+Those primitives are batch and layer normalizations, eltwise, prelu, lrn, shuffle and softmax.
+
+These operations, except shuffle, assume that only floating-point types will be used on input and output of same data type will be produced.
+It is quite essential due to the nature and service of these operations, especially, normalizations.
+Also, outputs of these operations are clearly distinguished only in floating-point range of values.
+
+Unfortunately, that assumption did not consider int8 flows when computations may contain mixture of quantized data and floating-point data.
+Mentioned primitives are still executed in floating-point data types such as `f32`, `bf16` or `f16`, but it usually happens that followed by primitive would use quantized output from mentioned primitives.
```

Comment:
I think "followed by primitive" should be "the following primitives" unless "followed by primitive" is a term I just haven't heard before.

Reply:
I like your suggestion, applied, thanks.